src/Apps/Consign/consignFromMyCollectionRoutes.tsx

```diff
@@ -127,6 +127,7 @@ export const consignFromMyCollectionRoutes: AppRouteConfig[] = [
     children: [
       {
         path: "/",
+        layout: "FullBleed",
```

Comment:
do we intentionally no longer want to show the footer? the sell page landing page is well optimized for SEO and could get us some installs

Reply:
good point! I add the footer [here](https://github.com/artsy/force/pull/13784/files#diff-d59a7e725761b99be4ef285de0b8dfc4266802ed45240ead40f7753850f01a8fR101) manually. The "FullBleed" layout is needed for the sticky footer (or better, how I've implemented the sticky footer...).

Reply:
thanks for explaining